java/com/google/location/lbs/activity/audioset/dataprocessing/incomplete-example.proto

```diff
@@ -0,0 +1,15 @@
+syntax = "proto3";
+
+option java_outer_classname = "IncompleteExample";
```

Comment:
could you elaborate what "Incomplete"Example means? sounds a bit off to me.

Reply:
Well, I was thinking of the protocol buffer being created from examples; however, I considered these examples incomplete as they do not have their corresponding features yet.